postgres/cds-plugin.js

```diff
@@ -1,9 +1,10 @@
-const cds = require('@sap/cds/lib')
+const cds = require('@sap/cds')
 
 if (!cds.env.fiori.lean_draft) {
   throw new Error('"@cap-js/postgres" only works if cds.fiori.lean_draft is enabled. Please adapt your configuration.')
 }
 
-if (cds.cli.command === 'build') {
-  module.exports = require('./lib/build')
-}
+cds.build?.register?.('postgres', {
+  impl: __dirname + '/lib/build.js',

```

Comment:
Is there a reason for using `__dirname + '/lib/build.js'` instead of `@cap-js/postgres/lib/build.js`?
`require` should correctly resolve...